nodeup/pkg/model/kubelet.go

```diff
@@ -145,6 +146,11 @@ func (b *KubeletBuilder) buildSystemdEnvironmentFile(kubeletConfig *kops.Kubelet
 		flags += " --network-plugin-dir=" + b.CNIBinDir()
 	}
 
+	// TODO how do we not hard code version??
```

Comment:
It's fine to do so, IMO.  We will need to update it per-k8s-version, but if that's the flag they give us, that's what we have to work with.